libs/blocks/global-navigation/global-navigation.js

```diff
@@ -615,6 +619,16 @@ class Gnav {
     });
   };
 
+  toggleMenuMobile = () => {
+    const toggle = this.elements.mobileToggle;
+    const isExpanded = toggle?.getAttribute('aria-expanded') === 'true';
```

Comment:
Same exact code is used on L284 & L656, might be worth turning it into a function
```
const isToggleExpanded = () => this.elements.mobileToggle?.getAttribute('aria-expanded') === 'true';
```